src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -18,7 +18,7 @@
 {%- endblock form_widget_simple -%}
 
 {%- block form_widget_compound -%}
-    <div {{ block('widget_container_attributes') }}>
+    <div{% if not form.methodRendered %} {{ block('widget_container_attributes') }}{% endif %}>
```

Comment:
After talking with @xabbuh, it seems that `form.methodRendered` is used to ensure that the _method field is not rendered twice, which is not what I am trying to do here.

Is there any way to know if a `form` is the `<form>` element?

Reply:
Perhaps you could try it with `form.parent is null`? Did not try, just an idea.

Reply:
It was my first attempt, using the rootform check. But if you do something like that:

```php
$form = $this->createForm(
    CollectionType::class,
    [
        'foo@example.com',
        'bar@example.com',
    ],
    [
        'entry_type' => EmailType::class,
        'entry_options' => [
            'attr' => ['class' => 'datetime-box'],
        ],
        'attr' => [
            'class' => 'foo',
            'data-foo' => 'attr',
        ],
        'row_attr' => [
            'class' => 'bar',
            'data-foo' => 'row-attr',
        ],
    ]);
```
And only calling `{{ form_widget(form) }}` in your template without `form_start`, then the rootform is the collection and it's ID and attribures will not be displayed.

Reply:
Making any change here looks like a BC break to me anyway as developers do probably rely on the id always being rendered here. IMO we would need a different id for the wrapping `<form>` element.

Reply:
How do you think this new form ID should be defined? Through a new option on the `FormType` or directly in the `form_start `block by concatenating for example `form_{{ id }}`?